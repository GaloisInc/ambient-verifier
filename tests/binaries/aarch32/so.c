#include "ambient_assert.h"
#include "fib.h"

int main(void) {
  ambient_assert(fib(6) == 8);
  return 0;
}

void _start(void) {
  main();
}
