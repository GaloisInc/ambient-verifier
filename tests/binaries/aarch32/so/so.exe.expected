successful: 9
failed: 0
overrideDir: tests/overrides/
sharedObjectsDir: tests/binaries/aarch32/so
